packages/lesswrong/components/tagging/AllTagsPage.tsx

```diff
@@ -74,7 +85,18 @@ const AllTagsPage = ({classes}: {
   return (
     <AnalyticsContext pageContext="allTagsPage">
       <div className={classes.root}>
-        <div className={classes.topSection}>
+        {isEAForum && <div className={classes.coreTagsSection}>
+          <ToCColumn
+            tableOfContents={<div/>}
+            header={<SectionTitle title={`Core ${taggingNamePluralSetting.get()}`} noTopMargin />}
+            centerEarly
+          >
+            <AnalyticsContext pageSectionContext="coreTagsSection">
```

Comment:
[Nit] Move this analytics context inside the `CoreTagsSection`. See this [underappreciated beauty of a slack message](https://cea-core.slack.com/archives/C75BWSNBH/p1673042350642009) for why the pattern of outside-the-component `AnalyticsContext`s emerged. Once we understand the better way to do things however, we realize that it's more sensible for them to be inside the component.